### PR TITLE
fix: script error is triggered when clicking the svg custom element

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* fixed that script error is triggered when clicking the svg custom element
* related issue(#1164)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/src/helper/dom.ts [hunk 1]
```diff
@@ -158,9 +158,17 @@ export function cls(...names: (ClassNameType | [boolean, ClassNameType])[]) {
   return result.join(' ');
 }
 
+function isSvgElement(el: Element) {
+  return el.namespaceURI?.indexOf('svg') !== -1;
+}
+
 export function isDatePickerElement(el: HTMLElement) {
   let currentEl: HTMLElement | null = el;
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/src/helper/dom.ts:166]: SVG가 넘어올 수도 있으니 `SVGElement` 타입도 추가되어야 하지 않을까요??

**Review by seonim-ryu — APPROVED** [review]
[9/17] 이견 없습니다. 👍

**Comment by 100milliongold** [comment]
감사합니다.

File: packages/toast-ui.grid/src/helper/dom.ts (hunk 1 above)
Comment (jung-han) [reply packages/toast-ui.grid/src/helper/dom.ts:166]: 위처럼 `Element`로 하면 되겠네요

File: packages/toast-ui.grid/types/renderer/index.d.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export type CellRendererProps = CellRenderData & {
 };
 
 export interface CellRenderer {
-  getElement(): HTMLElement;
+  getElement(): HTMLElement | SVGElement;
```
Comment (jung-han) [inline packages/toast-ui.grid/types/renderer/index.d.ts:18]: Element로 하는건 어떤가요?

**Review by jung-han — APPROVED** [review]
이견없습니다!

**Pull request merged**